### PR TITLE
Update comment format

### PR DESCRIPTION
**Pull request by LambStack**

### CODE REVIEW

**Comment by LambStack** [comment]
[//]: # (BOXPLOT)
**fusion-cli Benchmarks**
```diff
#These changes had no significant impact on the build time.
```
      
|[Help](https://docs.google.com/document/d/18TpXOEkjlfSVKfjdwJJZSe59Ju4U66sn4oH6egq1O84/edit?usp=sharing)|cached |uncached |incremental |production |
| ---------------- | ---------------- | ---------------- | ---------------- | ---------------- |
|∆ Median |-329 |-1595 |-259 |-1753 |
Median (head) |5006 |71898 |2804 |114898 |
IQR (head) |45 |485 |137 |431 |
Chart | [Boxplot](undefined/?json=%7B%22chartType%22%3A%22boxplot%22%2C%22title%22%3A%22cached%22%2C%22labels%22%3A%5B%22head%22%2C%22base%22%5D%2C%22data%22%3A%5B%5B4986%2C4994%2C5006%2C5039%2C5063%5D%2C%5B5126%2C5146%2C5335%2C5336%2C5435%5D%5D%7D) | [Boxplot](undefined/?json=%7B%22chartType%22%3A%22boxplot%22%2C%22title%22%3A%22uncached%22%2C%22labels%22%3A%5B%22head%22%2C%22base%22%5D%2C%22data%22%3A%5B%5B71871%2C71876%2C71898%2C72361%2C78712%5D%2C%5B71445%2C72508%2C73493%2C73532%2C79879%5D%5D%7D) | [Boxplot](undefined/?json=%7B%22chartType%22%3A%22boxplot%22%2C%22title%22%3A%22incremental%22%2C%22labels%22%3A%5B%22head%22%2C%22base%22%5D%2C%22data%22%3A%5B%5B2726%2C2753%2C2804%2C2890%2C2898%5D%2C%5B2987%2C3044%2C3063%2C3082%2C3293%5D%5D%7D) | [Boxplot](undefined/?json=%7B%22chartType%22%3A%22boxplot%22%2C%22title%22%3A%22production%22%2C%22labels%22%3A%5B%22head%22%2C%22base%22%5D%2C%22data%22%3A%5B%5B113994%2C114604%2C114898%2C115035%2C115983%5D%2C%5B115450%2C116387%2C116651%2C116865%2C118044%5D%5D%7D) |

File: probot-app-report-benchmarks/index.js [hunk 1]
```diff
@@ -124,8 +130,21 @@ module.exports = app => {
           labels: [`head`, `base`],
           data: [head_json.data[i].time, base_json.data[i].time],
         };
+        titles.push(head_json.data[i].build);
+        let headMedian = median(head_json.data[i].time);
+        let baseMedian = median(base_json.data[i].time);
+        let deltaMedian = headMedian - baseMedian;
+        if (Math.abs(deltaMedian) > baseMedian * 0.1) {
```
Comment (alxmyth) [inline probot-app-report-benchmarks/index.js:138]: Minor: can we move `0.1` into a named variable or add a comment?  Fairly important value that could use the additional context.

File: probot-app-report-benchmarks/index.js [hunk 2]
```diff
@@ -147,14 +166,62 @@ module.exports = app => {
...
+        tallyIncreased > 0 ? "-" : "+"
+      }These changes significantly ${
+        tallyIncreased > 0 ? "increased" : "decreased"
+      } the build time\n\`\`\``;
+    }
+    return `${commentString}\n**fusion-cli Benchmarks**\n${strEffect}
+      \n${createTableString(
+        titles,
+        ["∆ Median", "Median (head)", "IQR (head)", "Chart"],
+        [medianDeltas, medianDurations, iqrs, imageStrings]
```
Comment (alxmyth) [inline probot-app-report-benchmarks/index.js:193]: Should these have units associated with them?

**Comment by alxmyth** [comment]
This feels a lot more consumable!  Thanks for the changes.

File: probot-app-report-benchmarks/index.js [hunk 3]
```diff
@@ -188,3 +255,35 @@ module.exports = app => {
     );
   }
 };
+
+function createTableString(topTitles, sideTitles, data) {
+  let outString =
+    "|[Help](https://docs.google.com/document/d/18TpXOEkjlfSVKfjdwJJZSe59Ju4U66sn4oH6egq1O84/edit?usp=sharing)|";
```
Comment (LambStack) [inline probot-app-report-benchmarks/index.js:262]: Where should this link to? Right now its just the rfc for this in google docs, but perhaps there is a better location.

**Pull request merged**